core/src/tangram.h

```diff
@@ -130,7 +130,8 @@ enum DebugFlags {
     tile_infos,         // Debug tile infos
     labels,             // Debug label bounding boxes
     tangram_infos,      // Various text tangram debug info printed on the screen
-    all_labels          // Draw all labels
+    all_labels,         // Draw all labels
+    stats               // Frame states
```

Comment:
Typo: "Frame stats". I also find it nice to leave a trailing comma on the last entry in an enum.